src/events/cd-event-list.vue

```diff
@@ -65,6 +72,18 @@
       getSessionListForEvent(event) {
         return event.sessions.map(session => session.name).join(', ');
       },
+      isEventFull(eventId) {
+        const currentEvent = this.events.find(event => event.id === eventId);
```

Comment:
You can skip this step by just passing the whole `event` instead of the `eventId`.